### PR TITLE
docs: fix embedded view description issue

### LINKED ISSUE #46955: docs: TemplateRef.createEmbeddedView() mentions attaching the instantiated view to a view container
### Description

The TemplateRef.createEmbeddedView() method states:
>Instantiates an embedded view based on this template, and attaches it to the view container.

- What view container is this method description referencing? No `ViewContainerRef` is being used here.

I don't believe this description is correct, but please correct me if I am wrong.

### What is the affected URL?

https://angular.io/api/core/TemplateRef#createembeddedview

### PR DESCRIPTION
**Pull request by AlirezaEbrahimkhani**
createEmbeddedView method instantiates an embedded view based on this template and take your templateRef as argument and would render the passed templateRef in your ViewContainerRef.

resolves angular#46955

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/core/src/linker/template_ref.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export abstract class TemplateRef<C> {
 
   /**
    * Instantiates an embedded view based on this template,
-   * and attaches it to the view container.
+   * and take your templateRef as argument and would render the passed templateRef in your ViewContainerRef.
```
Comment (jessicajaniuk) [inline packages/core/src/linker/template_ref.ts:56]: This is a bit of a run on sentence that adds a bit too much. We don't really need to specify in the description here what the arguments are, as that's in the section just below this. I actually think the original text is cleaner here. Can you maybe take a second look here?

**Review by jessicajaniuk — DISMISSED** [review]

File: packages/core/src/linker/template_ref.ts (hunk 1 above)
Comment (Achilles1515) [reply packages/core/src/linker/template_ref.ts:56]: What about:

>Instantiates an **unattached** embedded view based on this template.

or 

>Instantiates an embedded view based on this template **that is unattached from a view container**.

Comment (jessicajaniuk) [reply packages/core/src/linker/template_ref.ts:56]: I like the first one better. It's clear and concise.

**Review by AndrewKushnir — APPROVED** [review]

**Comment by Achilles1515** [comment]
@AndrewKushnir @jessicajaniuk 
This should **not** be approved to merge.
The whole point of this PR was to remove the line about it attaching the view to a view container, because there is no view container.

**Comment by AndrewKushnir** [comment]
@Achilles1515 thanks for the comment. You are right, I misread the sentence, the "and attaches it to the view container" should be removed, so the description should say `Instantiates an embedded view based on this template.`.

**Comment by Achilles1515** [comment]
We were also going to add "unattached" to the description for increased clarification.
> Instantiates an **unattached** embedded view based on this template.

**Comment by jessicajaniuk** [comment]
@Achilles1515 Thanks for catching this before we merged it. I've replaced the fixup commit and updated with exactly that language.

**Review by AndrewKushnir — APPROVED** [review]

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit 5509e3512f9542896dbbc267752a5034be6df9f6.

**Pull request closed without merging**